packages/onchain-claims/package.json

```
@@ -20,7 +20,7 @@
     "compile": "npm run compile:contracts && npm run build",
     "compile:contracts": "truffle compile",
     "compile:ts": "bili",
-    "deploy:revocationregistryfactory": "node -r ts-node/register ./scripts/deploy_revocationRegistry",
+    "deploy:revocationregistryfactory": "node -r ts-node/register ./scripts/deploy-revocation-registry",
```

Script name is not precise as it deploys registry
```suggestion
    "deploy:revocationregistry": "node -r ts-node/register ./scripts/deploy-revocation-registry",
```